releases/stellar/2025-01-ITS-v1.0.0.md

```diff
@@ -0,0 +1,96 @@
+# STELLAR ITS v1.0.0
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @ahramy <ahram@interoplabs.io> |
+| **Deployment** | @RiceAndMeet <steven@interoplabs.io> |
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | TBD | TBD |
+| **Stagenet** | - | TBD |
+| **Testnet** | - | TBD |
+| **Mainnet** | - | TBD |
+
+- [Crates](https://crates.io/users/interoplabs-ci)
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases)
+
+## Background
+
+Changes in the release:
+
+1. This is the v1.0.0 initial ITS release.
+
+## Deployment
+
+Create an `.env` config.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=all
+```
+
+### Devnet Amplifier / Stagenet / Testnet / Mainnet
+
+```bash
+# Deploy Interchain Token
+node stellar/deploy-contract.js deploy interchain_token --version v1.0.0
+
+# Deploy token manager
+node stellar/deploy-contract.js deploy token_manager --version v1.0.0
+
+# Deploy Interchain Token Service
+node stellar/deploy-contract.js deploy interchain_token_service --version v1.0.0
+```
+
+## Checklist
+
+The following checks should be performed after the rollout
+
+### Stellar to EVM
+
+- [ ] Deploy Remote Token
+
+```bash
+node stellar/its.js deploy-interchain-token [symbol] [name] [decimals] [salt] [initial-supply]
+
+node stellar/its.js deploy-remote-interchain-token [salt] [destination-chain] [gas-token-address] [gas-fee-amount]
```

Comment:
avoid just dumping help text like cmds. add example inputs and detail info to ensure that the destination is executed and this token is transferred after